cypress/integration/page.spec.js

```diff
@@ -262,7 +291,7 @@
       });
 
       it('should not have users hardcoded', () => {
-        cy.intercept('**/users', { fixture: 'someUsers' })
+        cy.intercept('**/users', { fixture: 'someUsers' });
```

Comment:
There is a missing semicolon at the end of the `cy.intercept` statement. Ensure that each statement is properly terminated to avoid syntax errors.